src/guiguts/misc_tools.py

```diff
@@ -0,0 +1,130 @@
+"""Miscellaneous tools."""
+
+import logging
+import regex as re
+
+from guiguts.checkers import CheckerDialog, CheckerEntry
+from guiguts.maintext import maintext
+from guiguts.utilities import IndexRowCol, IndexRange
+
+logger = logging.getLogger(__package__)
+
+BLOCK_TYPES = "[$*XxFf]"
+POEM_TYPES = "[Pp]"
+
+
+def process_fixup(checker_entry: CheckerEntry) -> None:
+    """Process the fixup error."""
+    if checker_entry.text_range:
```

Comment:
If I've put an entire function within an if I like to negate the condition and return, then fall through to the actual code so I don't have to indent the whole function. Clearly just a personal preference and what you have works great, but I noticed it and thought I'd mention it.

Reply:
Oh yes, thanks! I found a couple other small things while fixing that too!